Calibration/HcalCalibAlgos/macros/CalibFitPlots.C

```diff
@@ -4142,7 +4148,8 @@ void PlotHistCorrRatio(char* infile1,
       int npt(0);
       for (std::map<int, cfactors>::const_iterator itr = cfacs[ih].begin(); itr != cfacs[ih].end(); ++itr) {
         int ieta = (itr->second).ieta;
-        if ((ieta >= etamin) && (ieta <= etamax) && ((itr->second).depth == depth1)) {
+        bool seleta = (etaMin > 0) ? (std::abs(ieta) > etaMin) : true;
+        if ((ieta >= etamin) && (ieta <= etamax) && seleta && ((itr->second).depth == depth1)) {
```

Comment:
The only difference with respect to the previous formulation is when `etamin>0`  and `etamin=ieta`, which was accepted with the previous formulation while it is rejected now.

In any case it does NOT correspond to what stated in the comment at the beginning of the file:
```C++
//      etaMin < 0 and etaMax > 0 will take ieta range from -etaMax to +etaMax;
//      etaMin > 0 will select ieta's where |ieta| is greater than etaMin
```

I think that what described in that comment should be coded as
```suggestion
        if ((ieta >= -etamax) && (ieta <= etamax) && seleta && ((itr->second).depth == depth1)) {
```

@bsunanda could you please check and confirm that this is exactly what you want (or fix that line if not)?